Cannot `make html`

This is what I get (things get interesting with WARNING, at ~80% down):

```
(python2.6.9)jo@jodel:~/Projekte/sympy-workspace/sympy-project/sympy/doc$ git status
Auf Branch improve-sysode-docs
nichts zu committen, Arbeitsverzeichnis unverändert
(python2.6.9)jo@jodel:~/Projekte/sympy-workspace/sympy-project/sympy/doc$ git checkout master
Zu Branch 'master' gewechselt
Ihr Branch ist auf dem selben Stand wie 'origin/master'.
(python2.6.9)jo@jodel:~/Projekte/sympy-workspace/sympy-project/sympy/doc$ make html
Makefile:121: warning: overriding recipe for target '_build/cheatsheet/cheatsheet.pdf'
Makefile:116: warning: ignoring old recipe for target '_build/cheatsheet/cheatsheet.pdf'
rm -rf _build/logo
mkdir -p _build/logo
python ./generate_logos.py -d
INFO:root: File saved: ./_build/logo/sympy-notail.svg
INFO:root: File saved: ./_build/logo/sympy-notailtext.svg
INFO:root: File saved: ./_build/logo/sympy-notext.svg
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-16px.png ./_build/logo/sympy-32px.png ./_build/logo/sympy-48px.png ./_build/logo/sympy-64px.png ./_build/logo/sympy-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notail-16px.png ./_build/logo/sympy-notail-32px.png ./_build/logo/sympy-notail-48px.png ./_build/logo/sympy-notail-64px.png ./_build/logo/sympy-notail-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notailtext-16px.png ./_build/logo/sympy-notailtext-32px.png ./_build/logo/sympy-notailtext-48px.png ./_build/logo/sympy-notailtext-64px.png ./_build/logo/sympy-notailtext-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notext-16px.png ./_build/logo/sympy-notext-32px.png ./_build/logo/sympy-notext-48px.png ./_build/logo/sympy-notext-64px.png ./_build/logo/sympy-notext-favicon.ico
DEBUG:root:return code: 0

Logo generated.
mkdir -p src/.static
mkdir -p _build/html
mkdir -p _build/doctrees
mkdir -p src/modules
PYTHONPATH=..: sphinx-build -b html -d _build/doctrees  src _build/html
Running Sphinx v1.3
WARNING: the config value 'html_domain_indices' has type `list', defaults to `bool.'
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 203 source files that are out of date
updating environment: 203 added, 0 changed, 0 removed
reading sources... [100%] wiki                                                                                      
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/core.rst:474: WARNING: error while formatting arguments for sympy.core.evalf.N: 'function' object has no attribute '__bases__'
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/functions/elementary.rst:306: WARNING: error while formatting arguments for sympy.functions.elementary.miscellaneous.sqrt: 'function' object has no attribute '__bases__'
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/polys/internals.rst:749: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_univariate: 'function' object has no attribute '__bases__'
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/polys/internals.rst:750: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_bivariate: 'function' object has no attribute '__bases__'
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/polys/internals.rst:751: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_multivariate: 'function' object has no attribute '__bases__'
/home/jo/Projekte/sympy-workspace/sympy-project/sympy/doc/src/modules/polys/internals.rst:752: WARNING: error while formatting arguments for sympy.polys.modulargcd.func_field_modgcd: 'function' object has no attribute '__bases__'
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... 
Theme error:
unsupported theme option 'visitedlinkcolor' given
Makefile:50: recipe for target 'html' failed
make: *** [html] Error 1
```

Interestingly, it works in my system.
```
Makefile:121: warning: overriding recipe for target '_build/cheatsheet/cheatsheet.pdf'
Makefile:116: warning: ignoring old recipe for target '_build/cheatsheet/cheatsheet.pdf'
rm -rf _build/logo
mkdir -p _build/logo
python ./generate_logos.py -d
INFO:root: File saved: ./_build/logo/sympy-notail.svg
INFO:root: File saved: ./_build/logo/sympy-notailtext.svg
INFO:root: File saved: ./_build/logo/sympy-notext.svg
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-160px.png -h 160 -w 160
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-500px.png -h 500 -w 500
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./src/logo/sympy.svg -f png -o ./_build/logo/sympy-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notail.svg -f png -o ./_build/logo/sympy-notail-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notailtext.svg -f png -o ./_build/logo/sympy-notailtext-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-16px.png -h 16 -w 16
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-32px.png -h 32 -w 32
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-48px.png -h 48 -w 48
DEBUG:root:return code: 0
DEBUG:root:command: rsvg-convert ./_build/logo/sympy-notext.svg -f png -o ./_build/logo/sympy-notext-64px.png -h 64 -w 64
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-16px.png ./_build/logo/sympy-32px.png ./_build/logo/sympy-48px.png ./_build/logo/sympy-64px.png ./_build/logo/sympy-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notail-16px.png ./_build/logo/sympy-notail-32px.png ./_build/logo/sympy-notail-48px.png ./_build/logo/sympy-notail-64px.png ./_build/logo/sympy-notail-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notailtext-16px.png ./_build/logo/sympy-notailtext-32px.png ./_build/logo/sympy-notailtext-48px.png ./_build/logo/sympy-notailtext-64px.png ./_build/logo/sympy-notailtext-favicon.ico
DEBUG:root:return code: 0
DEBUG:root:command: convert ./_build/logo/sympy-notext-16px.png ./_build/logo/sympy-notext-32px.png ./_build/logo/sympy-notext-48px.png ./_build/logo/sympy-notext-64px.png ./_build/logo/sympy-notext-favicon.ico
DEBUG:root:return code: 0

Logo generated.
mkdir -p src/.static
mkdir -p _build/html
mkdir -p _build/doctrees
mkdir -p src/modules
PYTHONPATH=..: sphinx-build -b html -d _build/doctrees  src _build/html
Running Sphinx v1.2.2
loading pickled environment... not yet created
building [html]: targets for 203 source files that are out of date
updating environment: 203 added, 0 changed, 0 removed
reading sources... [100%] wiki                                                                                                                     
/home/Nair/projects/sympy/doc/src/modules/core.rst:474: WARNING: error while formatting arguments for sympy.core.evalf.N: 'function' object has no attribute '__bases__'
/home/Nair/projects/sympy/doc/src/modules/functions/elementary.rst:306: WARNING: error while formatting arguments for sympy.functions.elementary.miscellaneous.sqrt: 'function' object has no attribute '__bases__'
/home/Nair/projects/sympy/doc/src/modules/polys/internals.rst:749: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_univariate: 'function' object has no attribute '__bases__'
/home/Nair/projects/sympy/doc/src/modules/polys/internals.rst:750: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_bivariate: 'function' object has no attribute '__bases__'
/home/Nair/projects/sympy/doc/src/modules/polys/internals.rst:751: WARNING: error while formatting arguments for sympy.polys.modulargcd.modgcd_multivariate: 'function' object has no attribute '__bases__'
/home/Nair/projects/sympy/doc/src/modules/polys/internals.rst:752: WARNING: error while formatting arguments for sympy.polys.modulargcd.func_field_modgcd: 'function' object has no attribute '__bases__'
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting                                              
writing output... [100%] wiki                                                                                                                      
writing additional files... (271 module code pages) _modules/index genindex py-modindex search
copying images... [100%] modules/physics/vector/kin_angvel2.svg                                                                                    
copying static files... done
copying extra files... done
dumping search index... done
dumping object inventory... done
build succeeded, 7 warnings.
cp -r src/pics _build/html/

Build finished. The HTML pages are in _build/html.

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -98,7 +98,7 @@ before_install:
   - if [[ "${TEST_SPHINX}" == "true" ]]; then
       sudo apt-get update;
       sudo apt-get install --no-install-recommends graphviz inkscape texlive texlive-xetex texlive-fonts-recommended texlive-latex-extra lmodern librsvg2-bin imagemagick docbook2x;
-      pip install "sphinx==1.1.3";
+      pip install "sphinx==1.3.1";
     fi
   - if [[ "${TEST_MATPLOTLIB}" == "true" ]]; then
       pip install "numpy==1.7.1";

--- a/doc/src/_static/default.css_t
+++ b/doc/src/_static/default.css_t
@@ -326,3 +326,12 @@ div.viewcode-block:target {
     border-top: 1px solid #ac9;
     border-bottom: 1px solid #ac9;
 }
+
+/*
+To highlight inline code blocks
+*/
+code{
+    background-color: #ECF0F3;
+    border-radius: 3px;
+    padding: 2px;
+}

--- a/doc/src/_static/default.css_t
+++ b/doc/src/_static/default.css_t
@@ -13,6 +13,23 @@
 
 /* -- page layout ----------------------------------------------------------- */
 
+{%  set theme_collapsiblesidebar = True %}
+{%  set theme_relbarbgcolor = '#2f441e'  %}
+{%  set theme_sidebarbgcolor = '#3b5526'  %}
+{%  set theme_sidebarbtncolor = '#4F663C'  %}
+{%  set theme_sidebarlinkcolor = '#81B953'  %}
+{%  set theme_linkcolor = '#29A329'  %}
+{%  set theme_visitedlinkcolor = '#307748'  %}
+{%  set theme_headtextcolor = '#2f441e'  %}
+{%  set theme_footerbgcolor = '#293b1b'  %}
+{%  set theme_headlinkcolor = '#AAAAAA'  %}
+{%  set theme_sidebartextcolor = '#DDDDDD'  %}
+{%  set theme_footertextcolor = '#DDDDDD'  %}
+{%  set theme_relbartextcolor = '#DDDDDD'  %}
+{%  set theme_relbarlinkcolor = '#81B953'  %}
+{%  set theme_bgcolor = '#FFFFFF'  %}
+
+
 body {
     font-family: {{ theme_bodyfont }};
     font-size: 100%;

--- a/doc/src/_static/default.css_t
+++ b/doc/src/_static/default.css_t
@@ -10,6 +10,7 @@
  */
 
 @import url("basic.css");
+@import url(http://fonts.googleapis.com/css?family=Open+Sans);
 
 /* -- page layout ----------------------------------------------------------- */
 
@@ -31,7 +32,7 @@
 
 
 body {
-    font-family: {{ theme_bodyfont }};
+    font-family: Open Sans, sans-serif;
     font-size: 100%;
     background-color: {{ theme_footerbgcolor }};
     color: #000;

--- a/doc/src/conf.py
+++ b/doc/src/conf.py
@@ -148,6 +148,7 @@ latex_elements = {
 \usepackage{bm}
 \usepackage{amssymb}
 \usepackage{fontspec}
+\usepackage[english]{babel}
 \defaultfontfeatures{Mapping=tex-text}
 \setmainfont{DejaVu Serif}
 \setsansfont{DejaVu Sans}

--- a/doc/src/conf.py
+++ b/doc/src/conf.py
@@ -96,22 +96,7 @@ html_last_updated_fmt = '%b %d, %Y'
 html_logo = '_static/sympylogo.png'
 html_favicon = '../_build/logo/sympy-notailtext-favicon.ico'
 # See http://sphinx-doc.org/theming.html#builtin-themes.
-html_theme_options = {
-    'collapsiblesidebar': True,
-    'relbarbgcolor': '#2f441e',
-    'sidebarbgcolor': '#3b5526',
-    'sidebarbtncolor': '#4F663C',
-    'sidebarlinkcolor': '#81B953',
-    'linkcolor': '#29A329',
-    'visitedlinkcolor': '#307748',
-    'headtextcolor': '#2f441e',
-    'footerbgcolor': '#293b1b',
-    'headlinkcolor': '#AAAAAA',
-    'sidebartextcolor': '#DDDDDD',
-    'footertextcolor': '#DDDDDD',
-    'relbartextcolor': '#DDDDDD',
-    'relbarlinkcolor': '#81B953',
-}
+
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.

--- a/doc/src/modules/core.rst
+++ b/doc/src/modules/core.rst
@@ -466,8 +466,7 @@ PrecisionExhausted
 
 N
 ^
-.. autoclass:: N
-   :members:
+.. autofunction:: N
 
 containers
 ----------

--- a/doc/src/modules/functions/elementary.rst
+++ b/doc/src/modules/functions/elementary.rst
@@ -303,9 +303,7 @@ Returns the square root of an expression. It is equivalent to raise to ``Rationa
     >>> sqrt(2) == 2**Rational(1,2)
     True
 
-.. autoclass:: sympy.functions.elementary.miscellaneous.sqrt
-   :members:
-
+.. autofunction:: sympy.functions.elementary.miscellaneous.sqrt
 
 sign
 ----

--- a/doc/src/modules/polys/internals.rst
+++ b/doc/src/modules/polys/internals.rst
@@ -746,10 +746,10 @@ Modular GCD
 
 .. currentmodule:: sympy.polys.modulargcd
 
-.. autoclass:: modgcd_univariate
-.. autoclass:: modgcd_bivariate
-.. autoclass:: modgcd_multivariate
-.. autoclass:: func_field_modgcd
+.. autofunction:: modgcd_univariate
+.. autofunction:: modgcd_bivariate
+.. autofunction:: modgcd_multivariate
+.. autofunction:: func_field_modgcd
 
 Manipulation of power series
 ****************************************************************************

